HI.json Hindi Language Support in Primon Proto

SamPandey001 opened the pull request.

phaticusthiccy commented:
Hi @SamPandey001 
Thank you for your pull request. Primon Proto continues to be developed as it is still in open beta. Many predetermined features have not been added yet in this process. Customizable language support is among these innovations. There are already codes that have been integrated into the bot but have been deprecated. 
Such innovations will come to Primon until the full version is released. 
Please check these codes.
https://github.com/phaticusthiccy/PrimonProto/blob/87f1a091f9e8418c8fd298b9526ac9474c699f77/lang.js#L222
https://github.com/phaticusthiccy/PrimonProto/blob/87f1a091f9e8418c8fd298b9526ac9474c699f77/qr.ts#L169

phaticusthiccy commented:
I'm thinking of adding only 2 languages ​​to Primon. English and Turkish. I added English because it is the world language and I added Turkish because it is my own language.

Apart from that, I thought of a feature for users;
Users will be able to modify the existing JSON file as they wish. Primon will explain how to do it with an example. Users will create their own language files by taking this example.

The codes that need to be correction, are available in Primon. I'll make it usable after a few tests and bug compiles. Please save this pull request for this feature. 

The main goal here is to reduce my burden. Since I know both existing languages ​​very well, I can develop quickly and conveniently. I know almost nothing about all other languages ​​(except german, azerbaijani and russian)

SamPandey001 commented:
> I'm thinking of adding only 2 languages ​​to Primon. English and Turkish. I added English because it is the world language and I added Turkish because it is my own language.
> 
> Apart from that, I thought of a feature for users; Users will be able to modify the existing JSON file as they wish. Primon will explain how to do it with an example. Users will create their own language files by taking this example.
> 
> The codes that need to be correction, are available in Primon. I'll make it usable after a few tests and bug compiles. Please save this pull request for this feature.
> 
> The main goal here is to reduce my burden. Since I know both existing languages ​​very well, I can develop quickly and conveniently. I know almost nothing about all other languages ​​(except german, azerbaijani and russian)

Ahh,That's Really Cool btw.
You WhatsAsenaDuplicated was great I loved that project.
Waiting for it's Release.
